docs/specification.md

```diff
@@ -38,8 +38,7 @@ BaseDAO is a concrete smart contract, but also a framework to implement various
 It can be configured at origination for any specific needs.
 
 In order to do so the contract has types that can contain arbitary data:
-- `proposal_metadata` which is a type synonym for `(string, bytes) map`
-  (or in Michelson: `map string bytes`)
+- `proposal_metadata` which is a type synonym for `bytes`
 - `contract_extra` which is a type synonym for `(string, bytes) big_map`
   (or in Michelson: `big_map string bytes`)
 
```

Comment:
A few lines below there's this:
```
For example a "treasury" style DAO would have a `proposal_metadata` containing:
```
which I think should be updated, as the `proposal_metadata` no longer contains `packed mutez` for each key, but the packed value for several fields. Can you update that example?

Reply:
I have updated this now, I believe `docs/registry.md` and `docs/treasury.md` are mainly for the Lorentz version, so some of the information are still out of sync if comparing to the ligo implementation.

Reply:
> are mainly for the Lorentz version

That should no longer be the case and they should be based on the Ligo version.
I see that you have updated them now and I think it's enough for this PR :ok_hand: